cmd/describe/autoscaler/cmd.go

```diff
@@ -0,0 +1,66 @@
+package autoscaler
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/clusterautoscaler"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	use     = "autoscaler"
+	short   = "Show details of the autoscaler for a cluster"
+	long    = short
+	example = ` # Describe the autoscaler for cluster 'foo'
+rosa describe autoscaler --cluster foo`
+)
+
+func NewDescribeAutoscalerCommand() *cobra.Command {
```

Comment:
The command is returned, fully initialised from a function. There is no use of global variables (ignoring the `output.AddFlag` and `ocm.AddClusterFlag` calls which are beyond the scope of this particular refactor). There is no use of `init`